pkg/autoscaler/collector_test.go

```diff
@@ -134,7 +134,7 @@ func TestMetricCollectorScraper(t *testing.T) {
 	logger := TestLogger(t)
 	ctx := context.Background()
 
-	now := time.Now()
+	now := time.Unix(0, 0)
```

Comment:
why this change?

Reply:
It helped me to satisfy myself that the test I added was working (since it gives me the same numbers for each test run). I'm not fussed about whether it stays or not.

Reply:
Actually, on making the change, I am fussed, the test begins to fail. I'm not immediately clear on why, I'll pick that up tomorrow.